app/src/main/java/com/flybits/conciergesample/GlobalApplication.kt

```diff
@@ -19,16 +20,19 @@ class GlobalApplication : Application() {
         Stetho.initializeWithDefaults(this)
         createNotificationChannel()
 
+
+
+        val config = FlybitsConciergeConfiguration.Builder(applicationContext)
+            .setGateWayUrl("https://api.demo.flybits.com")
+            .setProjectId("2CE41988-B1D3-4116-98DD-42FFB8754384")
+            .build()
+
+
         Concierge.setLoggingVerbosity(VerbosityLevel.ALL)
 
         //Call configure on Concierge
-        Concierge.configure(
-            null, arrayListOf(
-                ContextManager.PluginType.ReservedPlugin(
-                    ReservedContextPlugin
-                        .LOCATION
-                )
-            ), applicationContext
+        Concierge.configure(config, emptyList(),applicationContext)
+
```

Comment:
@mhollis1980FB hey do we want to configure using Builder becasue already have json connecting to sam project we can just pass null for the builder?